src/components/ContinuationIn/ContinuationAuthorization.vue

```diff
@@ -267,25 +272,23 @@ export default class ExtraproRegistration extends Mixins(DocumentMixin) {
    * This is called right after the File Validity event.
    */
   async onFileSelected (index: number, file: File): Promise<void> {
-    // reset state of file uploader to ensure not in manual error mode
-    this.fileUploadCustomErrorMsg = ''
```

Comment:
In the old code, this overwrote the error message set by the file upload component :(